commands/core/drupal/update.inc

```diff
@@ -130,6 +130,11 @@ function update_main() {
   drupal_load_updates();
   update_fix_compatibility();
 
+  // Check requirements before updating.
+  if (!drush_update_check_requirements()) {
```

Comment:
Maybe this should be a validation callback instead, so it's run before we hit the actual command? Then `drush_update_check_requirements ` can just be used directly or the validation command can call out to that as the code is now.